packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -668,6 +679,60 @@ class Auth {
         return await this.loginGuestUser()
     }
 
+    /**
+     * A wrapper method for commerce-sdk-isomorphic helper: authorizeIDP.
+     *
+     */
+    async authorizeIDP(parameters: Parameters<Helpers['authorizeIDP']>[1]) {
```

Comment:
In-lining the type for `parameters` as `Parameters<Helpers['authorizeIDP']>[1]` makes this function signature hard to read. Do you think creating an intermediate type with a readable name is possible?